### PR TITLE
feat: touch page

### PR DESCRIPTION
**Pull request by mayuran-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/features/pages/options/digital/data.ts [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    allOptionsMarkets.commodities,
+]
+
+export const upAndDownPlatforms: OptionContentAvailability[] = [
+    allOptionsTradePlatforms.deriv_trader,
+    allOptionsTradePlatforms.smart_trader,
+    allOptionsTradePlatforms.deriv_bot,
+]
+
+export const upAndDownFAQ: OptionsFAQDataItem[] = [
```
Comment (aswathy-deriv) [inline src/features/pages/options/digital/data.ts:64]: Data name should be `snakecase`

File: src/features/pages/options/digital/data.ts [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+    OptionContentAvailability,
+    OptionContentItemData,
+    OptionsFAQDataItem,
+} from 'features/components/templates/options-content/type'
+import {
+    allOptionsMarkets,
+    allOptionsTradePlatforms,
+} from 'features/components/organisms/available-markets-platforms/data'
+
+export const upAndDownContentItems: OptionContentItemData[] = [
```
Comment (aswathy-deriv) [inline src/features/pages/options/digital/data.ts:16]: Data name should be` snake_case`

File: src/features/pages/options/digital/data.ts
Comment (hasan-deriv) [inline src/features/pages/options/digital/data.ts:1]: Shouldn't it be inside touch-no-touch directory like [this](https://github.com/binary-com/deriv-com/blob/96d8683ef26446b0293535f4388f123afe4a7de4/src/features/pages/options/digital/up-down/data.ts)?

**Review by hasan-deriv — COMMENTED** [review]
Change the components and variables name from UpAndDown to TouchNoTouch, please

File: src/features/pages/options/digital/data.ts
Comment (mayuran-deriv) [reply src/features/pages/options/digital/data.ts:1]: yes man already working on it , forgot to change  , thanks

**Review by aswathy-deriv — APPROVED** [review]

File: src/features/pages/options/digital/touch-no-touch.tsx
Comment (hasan-deriv) [inline src/features/pages/options/digital/touch-no-touch.tsx:1]: Though, it doesn't matter. I still think so, we all should follow the same pattern for file/dir naming.

File: src/features/pages/options/digital/data.ts
Comment (hasan-deriv) [reply src/features/pages/options/digital/data.ts:1]: It still the same.

Comment (hasan-deriv) [reply src/features/pages/options/digital/data.ts:1]: I am just trying to follow michio's PR, if you think it's ok then we can go forward with this.

**Review by hasan-deriv — APPROVED** [review]

**Review by Nuzhy-Deriv — APPROVED** [review]

**Pull request merged**